trust-store-trust-policy-specification.md

```diff
@@ -158,41 +147,44 @@ The trust policy is represented as JSON data structure as shown below:
+- `permissive` : The `permissive` level enforces most signature verification checks, but will only logs failures for revocation and expiry. The `permissive` level is recommended to be used if signature verification is done at deploy time or runtime, and the user only needs integrity and authenticity guarantees.
+- `audit` : The `audit` level only enforces integrity check if a signature is present. Failure of all other checks are only logged.
+- `skip` : The `skip` level does not fetch signatures for artifacts and does not perform any signature verification. This is useful when an application uses multiple artifacts, and has a mix of signed and unsigned artifacts. Note that `skip` cannot be used with a global scope (`*`), the value of `registryScopes` MUST contain fully qualified registry URL(s).
+
+The following table shows the resultant behavior `enforced` (verification fails), or `logged` for each of the checks, based on signature verification level.
+
+|Signature Verification Level|Recommended Usage|Integrity|Authenticity|Trusted timestamp|Expiry|Revocation check|
+|----------------------------|-----------------|---------|------------|-----------------|------|----------------|
+|strict    |Use at development, build and deploy time|enforced|enforced|enforced|enforced|enforced|
+|permissive|Use at deploy time or runtime|enforced|enforced|logged|logged|logged|
+|audit     |Use when adopting signed images, without breaking existing workflows|enforced|logged|logged|logged|logged|
+|skip      |Use to exclude verification for unsigned images|skipped|skipped|skipped|skipped|skipped|
+
+**Integrity** : Guarantees that the artifact wasn't altered after it was signed, or the signature isn't corrupted. All signature verification levels always enforce integrity.
+
+**Authenticity** : Guarantees that the artifact was signed by an identity trusted by the verifier. Its definition does not include revocation, which is when a trusted identity is subsequently untrusted because of a compromise.
+
+**Trusted timestamp** : Guarantees that the signature was generated when the certificate was valid. It also allows a verifier to determine if a signature be treated as valid when a certificate is revoked, if the certificate was revoked after the signature was generated. In the absence of a trusted timestamp, signatures are considered invalid after certificate expires, and all signatures are considered revoked when a certificate is revoked. **NOTE**: `notation` RC1 will generate trusted timestamp using a TSA when signature is generated, but will not support verification of TSA countersignatures.
```

Comment:
<grammar> TSA when signature => TSA when the signature

Reply:
```suggestion
**Trusted timestamp** : Guarantees that the signature was generated when the certificate was valid. It also allows a verifier to determine if a signature be treated as valid when a certificate is revoked, if the certificate was revoked after the signature was generated. In the absence of a trusted timestamp, signatures are considered invalid after certificate expires, and all signatures are considered revoked when a certificate is revoked. **NOTE**: `notation` RC1 will generate trusted timestamp using a TSA when the signature is generated, but will not support verification of TSA countersignatures.
```